Tests/Unit/Helpers.swift

```diff
@@ -413,7 +413,7 @@ class MockMessageInterface: MiniAppMessageDelegate {
             id: "contact_id",
             name: MiniAppStore.randomFakeName(),
             email: MiniAppStore.fakeMail(with: MiniAppStore.randomFakeName()),
-            allEmailList: MiniAppStore.fakeMailList(with: MiniAppStore.randomFakeName())
+            allEmailList: ["test@test.com"]
```

Comment:
Why not testing with a list of more than 1 element? :)

Reply:
Thank you :) I have update this in new commit.